modules/system/assets/js/snowboard/winter-mix.js

```diff
@@ -0,0 +1,17 @@
+const mix = require('laravel-mix');
+require('laravel-mix-polyfill');
+
+mix.setPublicPath(__dirname);
+
+// Compile Snowboard framework
+mix
+    .js('./snowboard.base.js', './build/snowboard.base.js')
+    .js('./snowboard.base.debug.js', './build/snowboard.base.debug.js')
+    .js('./ajax/Request.js', './build/snowboard.request.js')
+    .js('./ajax/handlers/AttributeRequest.js', './build/snowboard.data-attr.js')
+    .js('./snowboard.extras.js', './build/snowboard.extras.js')
+    .polyfill({
+        enabled: mix.inProduction(),
```

Comment:
Does this mean that if Snowboard is compiled while app.debug = true (or some other indicator that the current .env is not production) then the polyfills won't be included? Wouldn't we want them to just always be included?

Reply:
This is more for framework development - `mix.inProduction()` is `true` if the `mix:compile` artisan command is run with the `production` flag. I don't include the polyfills in development because it slows down compilation, especially if I'm using `mix:watch` to watch for changes.

Reply:
gotcha, so I just need to remember to add the flag when committing new builds :)